Vagrantfile

```diff
@@ -75,8 +82,14 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   end
 
   # Parallels.
-  config.vm.provider :parallels do |p|
-    p.vm.box = "parallels/ubuntu-14.04"
+  config.vm.provider :parallels do |p, override|
```

Comment:
Same thing as above here—just use `vconfig['vagrant_box']`, and Parallels users will just have to specify their own box... maybe we can add a comment to that effect in config.yml, or I could just add a note in the Wiki somewhere.